README.md

```diff
@@ -1 +1,69 @@
-# kanban-board
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Movies Database
+
+> A webapp to find information about a collection of movies/shows. You can like your favorite shows, leave a commaent and make reservations for screening.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript ES6
+- Webpack
+- TvMaze API
+- Involvement API
+- Jest
+
+## Live Demo
+
+[Live Demo Link](https://drive.google.com/file/d/1GuBA_O6cSGohCIswKyfx4E8-lEnLcqyX/view?usp=sharing)
+
+## Video Presentation
+
+[Video Presentation Link](https://drive.google.com/file/d/1GuBA_O6cSGohCIswKyfx4E8-lEnLcqyX/view?usp=sharing)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps. If you would like to use a terminal you can use git clone and the project key: git@github.com:bengigo/movies-database.git
+
+### Prerequisites
+
+Web Browser (Chrome,Safari,Mozilla,Edge)
+
+### Install
+
+No need to install. Open the index.html file with a browser.
+
+### Run tests
+
+1- You need Jest loaded on your local environment.
+2- You can run test using `npm run test` command on your command line.
+
+## Authors
+
+👤 **Bengi Genç**
+
+- GitHub: [@githubhandle](https://github.com/bengigo)
+- Twitter: [@twitterhandle](https://twitter.com/bengi_gb)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/bengi-g-03b883199/)
+
+👤 **Michael Adetunji**
+
+- GitHub: [@githubhandle](https://github.com/Therealadetunji)
+- Twitter: [@twitterhandle](https://twitter.com/madetunji1376)
+
+👤 **Nonhlanhla Mndebele**
+
+- GitHub: [@githubhandle](https://github.com/29td)
+- Twitter: [@twitterhandle](https://twitter.com/NonhlanhlaMnde6)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/nonhlanhla-mndebele-ab7448226/)
+
+## 🤝 Contributing
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+-Thank you to Microverse Inc.
+
+## 📝 License
```

Comment:
- [ ] [OPTIONAL] Please kindly create an MIT license file and leave a link to it in the readme file 📃. You can see Microverse readme template for this 👍🏼